mobile/security.html

```diff
@@ -63,6 +63,9 @@ <h2>Technologies in progress</h2>
 
       <section class="featureset exploratory-work">
         <h2>Exploratory work</h2>
+        <div data-feature="Strengthened security">
+          <p>Cross-site scripting is considered one of the major attack vectors for the Web. <a data-featureid="trusted-types">Trusted Types</a> is an API that can be used to reduce the risk of DOM-based cross-site scripting by enforcing type constraints for DOM APIs.</p>
```

Comment:
First sentence is good. The description in the second sentence is factually correct, but leaves me with a "No idea what that spec does" feeling...

I find the sentence that appears in the introduction section of the spec to be somewhat clearer than the abstract and suggest to use it instead, illustrating the notion of "sinks" with a couple of examples:

```html
<p>Cross-site scripting (XSS) is considered to be one of the major attack vectors on the
Web. <a data-featureid="trusted-types">Trusted Types</a> allows applications to lock
down DOM-based XSS injection sinks  (e.g. <code>Element.innerHTML</code>, or
<code>Location.href</code> setters) to only accept non-spoofable, typed values in
place of strings.</p>
```